README.md

```diff
@@ -2,21 +2,49 @@
 
 [![PyPI version](https://badge.fury.io/py/google-api-python-client.svg)](https://badge.fury.io/py/google-api-python-client)
 
-This is the Python client library for Google's discovery based APIs. To get started, please see the [docs folder](docs/README.md).
+This is the legacy Python client library for Google's discovery based APIs.
+To get started, please see the [docs folder](docs/README.md).
 
-These client libraries are officially supported by Google.  However, the libraries are considered complete and are in maintenance mode. This means that we will address critical bugs and security issues but will not add any new features.
+The libraries are considered complete and are in maintenance mode. This means
+that we will address critical bugs and security issues but will not add any
+new features.
+
+This library is officially supported by Google.  However, the maintainers of
+this repository recommend using [Cloud Client Libraries for Python](https://github.com/googleapis/google-cloud-python),
+where possible, for new code development.
 
 ## Version 2.0 Release
 The 2.0 release of `google-api-python-client` is a significant upgrade compared
-to v1. Please see the [Migration Guide](UPGRADING.md) for more information.
+to v1. Please see the [Migration Guide](UPGRADING.md) for more information.  As
+a result of caching the discovery documents, the size of this package is at
+least 50 MB larger compared to the previous version.
 
 ## Documentation
 
 See the [docs folder](docs/README.md) for more detailed instructions and additional documentation.
 
 ## Other Google API libraries
 
-For Google Cloud Platform APIs such as Datastore, Cloud Storage or Pub/Sub, we recommend using [Cloud Client Libraries for Python](https://github.com/GoogleCloudPlatform/google-cloud-python).
+The maintainers of this repository recommend using
```

Comment:
Could you add a link to https://cloud.google.com/apis/docs/client-libraries-explained somewhere in this section?

Reply:
Done.